docs/concepts/copilot/copilot-ai-capabilities.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Data Copilot AI capabilities
+id: copilot-ai-capabilities
+description: The AI assistant capabilities for data Pipelines and models
+sidebar_position: 9
+tags:
+  - concepts
+  - copilot
+  - generativeai
+  - capabilities
+---
+
+Prophecy Data Copilot provides suggestions from an AI model as you develop your data Pipelines and Models. You can view and incorporate suggestions directly within the Prophecy visual editor and code editor. Data Copilot makes suggestions for your entire Pipeline, for a single Gem (transformation), and even for individual expressions within each Gem.
+
+## Supported AI capabilities by engine
+
+Data Copilot supports the following capabilites for the Spark and SQL engines:
+
+| AI Capability                                      | Spark                    | SQL                      |
+| -------------------------------------------------- | ------------------------ | ------------------------ |
+| Start a new Pipeline (Spark) or Model (SQL)        | ![Tick](./img/tick.svg)  | ![Tick](./img/tick.svg)  |
+| Modify an existing Pipeline (Spark) or Model (SQL) | ![Tick](./img/cross.svg) | ![Tick](./img/tick.svg)  |
+| Suggest Gems                                       | ![Tick](./img/tick.svg)  | ![Tick](./img/tick.svg)  |
+| Suggest Expressions                                | ![Tick](./img/tick.svg)  | ![Tick](./img/tick.svg)  |
+| Generate with AI, Scripts                          | ![Tick](./img/tick.svg)  | ![Tick](./img/cross.svg) |
+| Generate with AI Functions                         | ![Tick](./img/tick.svg)  | ![Tick](./img/tick.svg)  |
+| Map with AI                                        | ![Tick](./img/cross.svg) | ![Tick](./img/tick.svg)  |
+| Code with AI                                       | ![Tick](./img/cross.svg) | ![Tick](./img/tick.svg)  |
+| Fix with AI, Gems                                  | ![Tick](./img/cross.svg) | ![Tick](./img/tick.svg)  |
+| Fix with AI, Expressions                           | ![Tick](./img/tick.svg)  | ![Tick](./img/tick.svg)  |
+| Auto Documentation                                 | ![Tick](./img/tick.svg)  | ![Tick](./img/tick.svg)  |
+| Data Tests and Quality Checks                      | ![Tick](./img/tick.svg)  | ![Tick](./img/tick.svg)  |
+
+## Text to Pipelines
+
+Get started on a new Pipeline quickly by typing your query into the text box and Data Copilot will generate a new Pipeline or modify an existing one.
```

Comment:
by typing your prompt*

(Query makes people think about SQL query so let's use different terminology for user input. )

Reply:
Done.